docs/en/operations/system-tables/overview.md

```diff
@@ -114,7 +114,79 @@ WHERE `table` = 'pypi'
 
 Conversely, other system tables are node-specific e.g. in-memory or persisting their data using the MergeTree table engine. This is typical for data such as logs and metrics. This persistence ensures that historical data remains available for analysis. However, these node-specific tables are inherently unique to each node.
 
-To comprehensively view the entire cluster, users can leverage the [`clusterAllReplicas`](/sql-reference/table-functions/cluster) function. This function allows querying system tables across all replicas within the "default" cluster, consolidating node-specific data into a unified result. This approach is particularly valuable for monitoring and debugging cluster-wide operations, ensuring users can effectively analyze the health and performance of their ClickHouse Cloud deployment.
+In general, the following rules can be applied when determining if a system table is node-specific:
+
+- System tables with a `_log` suffix.
+- System tables that expose metrics e.g. `metrics`, `asynchronous_metrics`, `events`.
+- System tables that expose ongoing processes e.g. `processes`, `merges`.
+
+Additionally, new versions of system tables may be created as a result of upgrades or changes to their schema. These versions are named using a numerical suffix, where lower numbers generally indicate newer tables. The table without a suffix represents the latest version.
```

Comment:
> where lower numbers generally indicate newer tables

I doubt that.

Reply:
removed, its unclear we wont commit and reserve right to change either way